catalogs/catalog-hive/src/test/java/com/datastrato/gravitino/catalog/hive/integration/test/CatalogHiveIT.java

```diff
@@ -1024,7 +1024,7 @@ private void assertTableEquals(
 
     for (int i = 0; i < sortOrders.length; i++) {
       Assertions.assertEquals(
-          sortOrders[i].direction() == SortDirection.ASCENDING ? 0 : 1,
+          sortOrders[i].direction() == SortDirection.ASCENDING ? 1 : 0,
```

Comment:
We should use SQL to create a Hive table, then verify this by Gravitino to cover this PR's changes.

Reply:
@mchades, `SparkCommonIT#testCreateSortBucketTable` has already verified whether the `SparkTableInfo#getBucket` is equals for the changes.

Reply:
if we already have IT to verify this, why wasn't this issue caught before? 

I still suggest you add testing in CatalogHiveIT.

Reply:
@mchades, the expected value of `CatalogHiveIT` was wrong. We cannot conclude that this issue was not verified due to wrong expected values.

Reply:
Okay, I got it

Reply:
@mchades, the reason why this issue wasn't caught before is that the `SparkCommonIT` uses `SparkHiveTable` to check the bucket via Gravitino HiveTable instead of `HiveTable` of Spark, which should use `HiveTable` of Spark to check. Therefore when using HiveTable of Gravitino itself, this issue could not be verified.

Reply:
There is a Spark in CatalogHiveIT, can we create a Hive table by Gravitino client and then verify the sorting order using Spark SQL(such as `desc`, `show create table`) to make sure the semantic of sorting order is correct?